pipelines/toast_ground_sim.py

```diff
@@ -508,28 +135,16 @@ def parse_arguments(comm):
         '"fknee" (float, Hz), "alpha" (float), and '
         '"NET" (float).',
     )
-    parser.add_argument(
-        "--focalplane_radius",
-        required=False,
-        type=np.float,
-        help="Override focal plane radius [deg]",
-    )
     parser.add_argument(
         "--freq",
         required=True,
-        help="Comma-separated list of frequencies with " "identical focal planes",
-    )
-    parser.add_argument(
-        "--tidas", required=False, default=None, help="Output TIDAS export path"
-    )
-    parser.add_argument(
-        "--spt3g", required=False, default=None, help="Output SPT3G export path"
+        help="Comma-separated list of frequencies with identical focal planes",
```

Comment:
is `--freq` asking TOAST to only consider a subset of the frequencies available in the focalplane?

Reply:
No, the frequency is used for scaling the atmospheric signal. It overrides the bandpass read from the focal plane.  In the SO sims we use the actual bandpasses.

Reply:
can you add that to the argument description?

Reply:
Sure.